.gitignore

```diff
@@ -32,3 +26,44 @@ server/src/entities/__pycache__/Settings.cpython-39.pyc
 server/src/prestart/__pycache__/setenv.cpython-39.pyc
 server/src/shared/__pycache__/LoggerFactory.cpython-39.pyc
 server/src/utils/__pycache__/generatePKCE.cpython-39.pyc
+
+# Generic
+*.pyc
+*.komodoproject
+*~
+.cache/
+media/
+.swp
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Distribution / packaging
+.Python
```

Comment:
This ignore list seems big! Is there any way to do it more top level? Like an entire directory?

Reply:
It is large and we may be able to reduce it.  I will tackle this with any new functionality that we add.